### ISSUE TITLE
Colored Hearts

### ISSUE THREAD

**Issue by Zedrimar** [eidehua/More-Health-Enhanced#6]
For some reason, this mod references TiC's and DragonAPI's GUI images resulting in weird colored hearts instead of the normal heart GUI. Its minor but REALLY annoying!

**Comment by eidehua** [comment]
Have you tried custom-GUI = true and minimalistic-gui = false in the configs? 

And my code is based on minecraft's hearts, I don't reference other mod's guis. It can be the case the other mods are doing something to change how minecraft renders hearts, and that affects mine.

**Comment by Zedrimar** [comment]
yes, in fact every setting config that I have tried still resulted in your mod for some reason seeking out dragonAPI's GUI (dragonAPI does do some "hacky" things though).
